src/main/java/automation/app/ButtonsPage.java

```diff
@@ -0,0 +1,14 @@
+package automation.app;
+
+import automation.base.BasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ButtonsPage extends BasePageObject
+{
+    final public WebElement buttonClickMe = this.wd().findElement(By.xpath("//div[@class=\"mt-4\"][2]/button"));
```

Comment:
You can use `'` instead of `"` in xpath. For example:
`"//div[@class=\"mt-4\"][2]/button"`
could be replaced with
`"//div[@class='mt-4'][2]/button"`
it has better readability